### ISSUE TITLE
Allow kata-check to check for latest version

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/kata-containers#734]
TL;DR - add a new `kata-check` test to see if there is a newer version of Kata available for download.

For Kata 2.0 we are planning to minimise custom packaging, instead relying on distro packaging [[1]][[2]]. However, we are still going to provide static binaries on the release page [[3]]. The static binaries are extremely useful for cloud deployments since you just unpack the archive somewhere and run. But that suggests users may start using these as a convenience too (particularly if they want the latest-and-greatest release [distro packaging will inevitably lag behind a little]). Therein lies a problem - if users are installing Kata without a package manager, the installation cannot be automatically updated. We can't solve that problem but what we can do is provide a way for users to determine if there *is* a newer vesrsion for them to download.

[1]: https://github.com/kata-containers/packaging/issues/1120
[2]: https://github.com/kata-containers/documentation/issues/708
[3]: https://github.com/kata-containers/runtime/releases
